### PR TITLE
8: Create top menu

### PR DESCRIPTION
**Pull request by ducmanh2111**
https://ruby-training.tk/issues/8

### CODE REVIEW

File: app/views/layouts/application.html.slim [hunk 1]
```diff
@@ -8,6 +8,14 @@ html
     = stylesheet_link_tag 'application', media: 'all'
     = javascript_pack_tag 'application'
   body.sidebar-lg-show[data-controller="#{controller_path}" data-action="#{action_name}"]
+    nav
```
Comment (ngocnguyenbk) [inline app/views/layouts/application.html.slim:11]: Viết riêng ra partial.

**Comment by ducmanh2111** [comment]
image

File: app/views/layouts/application.html.slim (hunk 1 above)
Comment (ducmanh2111) [reply app/views/layouts/application.html.slim]: OK

File: app/assets/stylesheets/_menu.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+nav a:hover {
+  text-decoration: none;
+}
+
+nav .is-active {
+  text-decoration: underline;
```
Comment (ngocnguyenbk) [inline app/assets/stylesheets/_menu.scss:6]: Nên thay bằng `font-weight: bold;` sẽ đẹp hơn.

Comment (ducmanh2111) [reply app/assets/stylesheets/_menu.scss]: OK

**Comment by ngocnguyenbk** [comment]
LGTM

**Pull request merged**